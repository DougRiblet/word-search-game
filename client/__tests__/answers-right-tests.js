import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import AnswersRight from '../components/Answers-Right/answers-right-present';

describe('Answers Right', () => {
  test('Snapshot', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const AnswersRight_tree = renderer.create(
      <AnswersRight foundfour={dummystring} />
    ).toJSON();
    expect(AnswersRight_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const div = document.createElement('div');
    ReactDOM.render(<AnswersRight foundfour={dummystring} />, div);
  });
});
