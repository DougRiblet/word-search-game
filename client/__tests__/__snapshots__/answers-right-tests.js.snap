// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answers Right Snapshot 1`] = `
<div
  id="answers-right"
>
  ABCD EFGH MNOP
</div>
`;
